src/styles/blocks/clock.scss

```diff
@@ -0,0 +1,50 @@
+.clock {
+  @include element-size(500px);
+  border: 4px solid red;
+  border-radius: 50%;
+
+  &__center {
+    @include element-size(20px);
+    position: relative;
+  }
+
+  &__minute-hand {
+    height: 210px;
+    width: $width-hand;
+    position: absolute;
+    left: 50%;
+    top: 55px;
+    background-color: $hands-color;
+    animation-name: move;
+    animation-duration: 90s;
+    animation-iteration-count: infinite;
+    transition: transform;
+    transform-origin: bottom;
+    transition-timing-function: linear;
```

Comment:
you repeat this part of code twice, only with different duration, create mixin for it